docs/user/ppl/cmd/search.rst

```diff
@@ -62,3 +62,274 @@ PPL query::
+
+    os> search age>30 AND age<=35 source=accounts;
+    fetched rows / total rows = 2/2
+    +----------------+-----------+----------------------+---------+--------+--------+----------+-------+-----+----------------------+----------+
+    | account_number | firstname | address              | balance | gender | city   | employer | state | age | email                | lastname |
+    |----------------+-----------+----------------------+---------+--------+--------+----------+-------+-----+----------------------+----------|
+    | 1              | Amber     | 880 Holmes Lane      | 39225   | M      | Brogan | Pyrami   | IL    | 32  | amberduke@pyrami.com | Duke     |
+    | 18             | Dale      | 467 Hutchinson Court | 4180    | M      | Orick  | null     | MD    | 33  | daleadams@boink.com  | Adams    |
+    +----------------+-----------+----------------------+---------+--------+--------+----------+-------+-----+----------------------+----------+
+
+Character Escaping and Phrase Search
+=====================================
+
+Phrase Search with Double Quotes
+---------------------------------
+Double quotes in search values indicate phrase search:
+
+* ``search address="Bristol Street" source=accounts`` 
+* Expected output: Documents where address field contains the exact phrase "Bristol Street"
+
+Special Characters
+------------------
+The following Lucene special characters are automatically escaped in values:
+``+ - && || ! ( ) { } [ ] ^ " ~ * ? : \ /``
+
+Examples:
+
+* ``search email="user+test@example.com" source=accounts``
+  * Expected output: Documents where email contains the literal text "user+test@example.com"
+* ``search address="123 Main St. (Suite #4)" source=accounts``
+  * Expected output: Documents where address contains the literal text "123 Main St. (Suite #4)"
+
+Field Names
+-----------
+* Dots (``.``) in field names are preserved for nested fields
+* Example: ``user.name`` accesses the nested field
+
+Important Notes
+===============
+
+Wildcard Pattern Matching
+-------------------------
+Wildcards are supported for pattern matching in search expressions:
+
+* ``*`` - Matches zero or more characters
+* ``?`` - Matches exactly one character
+
+**Wildcard Examples:**
+
+* ``search severityText=ERR* source=logs`` - Matches ERROR, ERROR2, ERROR3, etc.
+* ``search name=John? source=accounts`` - Matches Johns, Johny, etc. (exactly 5 characters)
+* ``search email=*@example.com source=accounts`` - Matches any email ending with @example.com
+* ``search fail* source=logs`` - Free text search for words starting with "fail"
+
+**Escaping for Literal Matching:**
+
+To search for literal ``*``, ``?``, or ``\`` characters, you must escape them:
+
+.. list-table:: 
+   :widths: 30 35 35
+   :header-rows: 1
+
+   * - Search For
+     - PPL Syntax
+     - Example
+   * - ``prefix*`` (wildcard)
+     - ``field=prefix*``
+     - Matches "prefix", "prefixABC", etc.
+   * - ``prefix*`` (literal)
+     - ``field="prefix\\*"``
```

Comment:
Previous section callout the following Lucene special characters are automatically escaped in values. Is it conflict.